app/src/main/res/drawable/tab_indicator.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:end="52dp"
```

Comment:
You shouldn't use constant size.
Always look for a solution that will work on all screens.
By simply googling "android tablayout indicator center" I found that you can add
`
android:gravity="center_horizontal"
`
and it will center the indicator.
So please remove the end, start and use that attribute.